src/components/event/MapModal/Map-modal.js

```diff
@@ -84,19 +84,19 @@ export const MapModal = ({ initialize, values }) => {
             </div>
           )}
           <div>
-            {!!values.selectedPos.latitude && !!values.selectedPos.longitude && (
+            {latitude && longitude && (
```

Comment:
I see that this condition is same as on 99 line. Can it be one conditional render instead of two ones